cartesian_enigma/rust/src/main.rs

```diff
@@ -0,0 +1,20 @@
+#[cfg(test)]
+#[path = "./cartesian_test.rs"]
+mod cartesian_test;
+
+fn main() {
+    let args: Vec<String> = std::env::args().collect();
+    if args.len() != 2 {
+        return;
+    }
+    let input = args.get(1).unwrap();
+
+    let message = cartesian_decoder(input);
+    println!("\n{}", message);
+
+}
+
+fn cartesian_decoder(mut input: &String) -> String {
```

Comment:
`mut` is not necessary, it will meant that we pass a mutable point to the String to function.

Reply:
Nice catch, it was left here from a previous try to code the decoder differently